src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -125,6 +125,15 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 		startupCommand = common.ExtendPathForCommand(startupCommand, gen.SourcePath)
 	}
 
+	scriptBuilder.WriteString("if [ -f ./" + common.ManifestFileName + " ]; then\n")
+	scriptBuilder.WriteString("    isInManifest=$(cat " + common.ManifestFileName + " | grep -c 'NODE_OPTIONS')\n")
+	scriptBuilder.WriteString("    if [ $isInManifest -eq 0 ]; then\n")
+	scriptBuilder.WriteString("        echo \"Oryx has set up Code-less App-Insight...\"\n")
+	scriptBuilder.WriteString("        export NODE_OPTIONS='--require ./oryxappinsightloader.js'\n")
```

Comment:
we should also append existing values of $NODE_OPTIONS the user might have set as an app setting